mcs/class/corlib/darwin_net_4_x_corlib.dll.exclude.sources

```diff
@@ -0,0 +1 @@
+../../../external/corert/src/Common/src/Interop/Windows/mincore/Interop.MemAllocFree.cs
```

Comment:
Move Windows/mincore/Interop.MemAllocFree.cs to win32_net_4_x_corlib.dll.sources and you can avoid all excludes.sources